components/ordhook-cli/src/config/file.rs

```diff
@@ -49,7 +49,7 @@ impl ConfigFile {
             _ => return Err("network.mode not supported".to_string()),
         };
 
-        let snapshot = match config_file.snapthot {
+        let snapshot = match config_file.snapshot {
             Some(bootstrap) => match bootstrap.download_url {
```

Comment:
Does this need to be changed to `snapshot` rather than `bootstrap` too? Sorry, I dont know Rust

Reply:
Correct!

Reply:
ah you mean the inner field? that's not required

Reply:
haha, idk, I just changed it all to snapshot as the bootstrap word looked like it was being phased out. I seems like it is working here.